src/main/webapp/components/NotePreview.vue

```diff
@@ -77,6 +82,39 @@
         fetch('/download-note?note_id=' + this.id, {
           method: 'POST'
         })
+      },
+      toggleFavorite: function() {
+        this.getFavorited()
+          .then(isFavorited => {
+            this.favorited = isFavorited;
+            const method = this.favorited ? 'DELETE' : 'POST';
+            fetch('/favorite-note?note_id=' + this.id, {
+              method: method
+            }).then(response => {
+              if(response.status === 403) alert('Please sign in to favorite this note.');
+              else this.favorited = !this.favorited;
+            })
+          })
+      },
```

Comment:
Looks like NoteGridCard.vue is still using the old toggleFavorite function. Could you add that change to this PR?

Reply:
I'm not sure where you see that NoteGridCard.vue is using the old toggleFavorite function. Is it in this PR?

Reply:
No, but I made these changes in #73 , so should be good to go :)

Reply:
Ok thanks!